File: 351003/Razhnou/task340/publisher/src/main/java/by/bsuir/task340/publisher/dto/response/ReactionResponseTo.java
package by.bsuir.task340.publisher.dto.response;

import by.bsuir.task340.publisher.dto.ReactionState;
import com.fasterxml.jackson.annotation.JsonProperty;

public record ReactionResponseTo(
        Long id,
        @JsonProperty("tweetId")
        Long tweetId,
        String content,
        ReactionState state
) {
}

File: 351003/Gorodko/app/Publisher/Dto/ReactionResponseTo.cs
using System.Text.Json.Serialization;

namespace Publisher.Dto {
    public class ReactionResponseTo : BaseResponseTo {
        [JsonPropertyName("tweetId")]
        public long TweetId { get; set; }

        [JsonPropertyName("content")]
        public string Content { get; set; } = string.Empty;
    }
}